website/scripts/generate-graphql-schema.sh

```diff
@@ -0,0 +1,18 @@
+mkdir -p ./website/schemas
```

Comment:
add shebang and `set -e`

Reply:
462dd558c674242483abd8f8ef865aafe2576ed7